monitoring/src/main/resources/web/js/init.js

```diff
@@ -1,17 +1,18 @@
 
     function frontendApp() {
-        return akka.viz.frontend.FrontendApp()
+        return akkaviz.frontend.FrontendApp()
     }
 
     var graph = Viva.Graph.graph();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["Viva" is not defined.](https://www.codacy.com/app/lustefaniak/akka-viz/file/2099574923/issues/source?bid=3121282&fileBranchId=3142760#l6)